README.md

```diff
@@ -97,37 +95,44 @@ PresentationModel instance is automatically retained during configuration change
 Lifecycle callbacks:
 - `onCreate()` — Called when the PresentationModel is created. Initialize your Rx chains in this method.
 - `onBind()` — Called when the View binds to the PresentationModel.
+- `onResume` - Called when the View resumes and begins to receive updates from states and commands.
+- `onPause` - Called when the View pauses. At this point, states and commands stops emitting to the View and turn on internal buffer until the View resumes again. 
```

Comment:
Called when the View pauses. At this point, states and commands stop emitting to the View and turn on internal buffer until the View resumes again.